peps/converters.py

```diff
@@ -180,9 +152,6 @@ def convert_pep_page(pep_number, content):
     pep_content.body.unwrap()
 
     data['content'] = str(pep_content)
-
-    source_link = "https://github.com/python/peps/blob/master/pep-{0}.txt".format(pep_number)
```

Comment:
Did you move this inside `get_pep_page()` because you called `convert_pep_page()` in `convert_pep0()`, right?

Reply:
Yes.